openstack/antiddos/v1/antiddos/testing/fixtures.go

```diff
@@ -14,8 +14,8 @@ import (
 const CreateOutput string = `
 {
   "error_code": "10000000",
-  "error_description": "The task has been received and is being handled",
-  "task_id": "82463800-70fe-4cba-9a96-06175e246ab3"
+  "error_msg": "Ok",
+  "task_id": ""
```

Comment:
Here and further: why those test values were changed?

Reply:
According to the new API (our docs is outdate). It's should be change.